sway/tree/container.c

```diff
@@ -548,7 +518,6 @@ void container_begin_destroy(struct sway_container *con) {
 	container_end_mouse_operation(con);
 
 	con->node.destroying = true;
```

Comment:
Do we still use `destroying` for anything?

Reply:
I don't think so no - I have removed both it and `view->destroying` now.